azurerm/internal/services/loganalytics/log_analytics_workspace_resource.go

```diff
@@ -91,6 +103,24 @@ func resourceArmLogAnalyticsWorkspace() *schema.Resource {
 				Deprecated: "this property has been removed from the API and will be removed in version 3.0 of the provider",
 			},
 
+			"private_link_scoped_resource": {
```

Comment:
what value would surfacing this field have? it doesn't appear to be useful?

Reply:
once a workspace is associated to an azure monitor private link scope the `private_link_scoped_resource property` will have values, and then users can get the id of associated private link scope from it.
An example of `private_link_scoped_resource property` in response from service is:
```
            {
                "resourceId": "/subscriptions/7fd08dcc-a653-4b0f-8f8c-4dac889fdda4/resourceGroups/weidongtest/providers/microsoft.insights/privatelinkscopes/dwtest2/scopedresources/scoped-dwtest-fedbdf3a-0400-423c-99d7-7fa2284d6055",
                "scopeId": "daaef2c4-45a2-479a-9d6b-84fc5b06d09c"
            },
            {
                "resourceId": "/subscriptions/7fd08dcc-a653-4b0f-8f8c-4dac889fdda4/resourceGroups/weidongtest/providers/microsoft.insights/privatelinkscopes/dwtest/scopedresources/scoped-dwtest-fedbdf3a-0400-423c-99d7-7fa2284d6028",
                "scopeId": "ddbcb8fe-259d-4682-9dd3-a78a15db5772"
            }
```
thanks

Reply:
@dw511214992 apologies, missed this reply - in this instance since this isn't available at the time the resource is created - and is related to the private link, this shouldn't be exposed on this resource. Instead this field wants to be exposed on the Private Link Endpoint resource, since that information is only available (e.g. populated) once that resource has been created.